src/Kentico.Xperience.Lucene.Admin/Client/src/entry.tsx

```diff
@@ -1 +1,2 @@
 export * from './templates';
+export * from './models';
```

Comment:
Is it necessary to export everything? Shouldn't be exported just form component?